realm/realm-library/src/main/cpp/util_sync.hpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef REALM_UTIL_SYNC_HPP
+#define REALM_UTIL_SYNC_HPP
+
+#include <jni.h>
+#include <util/bson/bson.hpp>
+
+realm::bson::Bson jstring_to_bson(JNIEnv* env, jstring arg);
```

Comment:
These functions are currently free functions. I would probably put them inside a class. `BsonUtil` perhaps.

Reply:
Ok. Will take it into account for later updates.